sdk/search/search-documents/review/search-documents.api.md

```diff
@@ -1131,6 +1121,63 @@ export interface SearchDocumentsResultBase {
     };
 }
 
+// @public
+export class SearchIndexClient {
+    constructor(endpoint: string, credential: KeyCredential, options?: SearchIndexClientOptions);
+    analyzeText(indexName: string, options: AnalyzeTextOptions): Promise<AnalyzeResult>;
+    readonly apiVersion: string;
+    createIndex(index: Index, options?: CreateIndexOptions): Promise<Index>;
+    createOrUpdateIndex(index: Index, options?: CreateOrUpdateIndexOptions): Promise<Index>;
+    createOrUpdateSynonymMap(synonymMap: SynonymMap, options?: CreateOrUpdateSynonymMapOptions): Promise<SynonymMap>;
+    createSynonymMap(synonymMap: SynonymMap, options?: CreateSynonymMapOptions): Promise<SynonymMap>;
+    deleteIndex(index: string | Index, options?: DeleteIndexOptions): Promise<void>;
+    deleteSynonymMap(synonymMap: string | SynonymMap, options?: DeleteSynonymMapOptions): Promise<void>;
+    readonly endpoint: string;
+    getIndex(indexName: string, options?: GetIndexOptions): Promise<Index>;
+    getIndexStatistics(indexName: string, options?: GetIndexStatisticsOptions): Promise<GetIndexStatisticsResult>;
+    getSearchClient<T>(indexName: string, options?: SearchClientOptions): SearchClient<T>;
```

Comment:
It seems a little strange to me that we have `getSearchClient` here, but no link from the search client to the index or indexer clients. Also there doesn't seem to be `getSearchClient` on the indexer client?

I think we should remove this unless we want to add more methods to move between clients, and if we want to do that, we shouldn't have it return the direct client class, but rather an interface that describes the shape of the class (to avoid circular imports)

Reply:
Agreed & Removed.